.github/workflows/timezone-update.yml

```diff
@@ -34,12 +34,22 @@ jobs:
       - name: Record new version
         run: echo "new_version=$(ls icu-data/tzdata/icunew | tail -1)" >> $GITHUB_ENV
 
+      - name: Record current version
+        run: echo "current_version=$(node -e 'console.log(process.versions.tz)')" >> $GITHUB_ENV
```

Comment:
This won't work/be accurate -- we don't have a built version of `node` here (or if there is one it hasn't been built from the checked out code).

Reply:
Thats right, is there a file in which the version is written? I remember being a .txt

Reply:
it seems there is test/fixtures/tz-version.txt, it can be used to check?